modules/claims_api/app/swagger/claims_api/v2/production/swagger.json

```diff
@@ -1912,6 +1912,20 @@
                                     "default": false,
                                     "nullable": true
                                   },
+                                  "specialIssues": {
+                                    "type": "array",
```

Comment:
@rockwellwindsor-va Since the special Issues are not ready for release into prod, I deleted that attribute in production/swagger, it is in 4 places.

Reply:
Thanks for the catch on that!